src/scripts/main.js

```diff
@@ -1,3 +1,230 @@
 'use strict';
 
-// write code here
+const tableBody = document.body.querySelectorAll('TBODY')[0];
+const tableHead = document.querySelector('THEAD');
+
+let sortFlow = 1;
+
+function sortAlgorythm(e) {
+  const rows = [...tableBody.children];
+
+  let index;
+
+  switch (e.target.textContent) {
+    case 'Name':
```

Comment:
Better not to hardcode the column name, as it can change, but to check if the name includes '$' and can be converted to a number.